notebooks/278-kosmos2-multimodal-large-language-model/278-kosmos2-multimodal-large-language-model.ipynb

```diff
@@ -0,0 +1,897 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #36.    print(processed_text)
</pre><p>can some additional text prints will be added for explanation which info each output provide.</p><p>Fo boxes for example, it can be drown on picture (it is hard to match input data and coordinates location), and also the fact that you prints generated text before and after postprocessing can be not clear for user</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1483/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
Extra information and drawing pictures with boxes were added